<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Face_Frame</class>
 <widget class="QWidget" name="Face_Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="Face_QLabel" name="label_1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_5">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_6">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_7">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_8">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="Face_QLabel" name="label_9">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_10">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_11">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_12">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_13">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_14">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_15">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_16">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="Face_QLabel" name="label_17">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_18">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_19">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_20">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_21">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_22">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_23">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_24">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="Face_QLabel" name="label_25">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_26">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_27">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_28">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_29">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_30">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_31">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_32">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="Face_QLabel" name="label_33">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_34">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_35">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_36">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_37">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_38">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_39">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_40">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="Face_QLabel" name="label_41">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_42">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_43">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_44">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_45">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_46">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_47">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_48">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="Face_QLabel" name="label_49">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_50">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_51">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_52">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_53">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_54">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_55">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_56">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="Face_QLabel" name="label_57">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_58">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_59">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_60">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_61">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_62">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_63">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Face_QLabel" name="label_64">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Face_QLabel</class>
   <extends>QLabel</extends>
   <header location="global">face_qlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
